reana/cli.py

```diff
@@ -1647,12 +1668,35 @@ def cluster_build():  # noqa: D301
 
 
 @cli.command(name="cluster-deploy")
-def cluster_deploy():  # noqa: D301
+@click.option(
+    "--namespace", "-n", default="default", help="Kubernetes namespace [default]"
+)
+@click.option(
+    "-d",
+    "--debug",
+    is_flag=True,
+    default=False,
+    help="Should we deploy REANA in production mode? ",
+)
+def cluster_deploy(namespace, debug):  # noqa: D301
     """Deploy REANA cluster."""
-    for cmd in [
-        "helm install reana helm/reana -f helm/configurations/values-dev.yaml -n default --wait",
+    dev_values_yaml = "helm/configurations/values-dev.yaml"
+    debug_flags = "--set debug.enabled=true" if debug else ""
+    helm_install = "helm install reana helm/reana -n {namespace} -f {values} --create-namespace {debug_flags} --wait".format(
+        namespace=namespace, values=dev_values_yaml, debug_flags=debug_flags
+    )
+    deploy_cmds = [
+        helm_install,
+        "kubectl config set-context --current --namespace={}".format(namespace),
+        "sleep 5",  # Let the database initialise.
```

Comment:
@tiborsimko I had to add this as sometimes the `create-admin-user.sh` would fail with an OperationalError trying to connect to Postgres which was still being initialised, did you see this problem? The correct/long-term solution for this would be to add `livenessProbes` to PosgreSQL, or even better, use the [PostgreSQL Helm Chart](https://hub.helm.sh/charts/bitnami/postgresql/3.18.3) so we don't maintain the template.

Reply:
Haven't seen this with the "old kind" times; using `--wait` was always enough there.  We could make a mini sleep loop as in Makefile times until we go full pg helm...

Reply:
Added a new tiny script `wait-for-db.sh`. I am totally flexible, it can be renamed or even, this logic could be added inside `create-admin-user.sh` as we do not need it anywhere else.

Reply:
Hmm, do we need an "external" shell script?  Perhaps doing the loop inside `reana-dev` Python code in this very place may be enough? 

Yeah, adding it to `create-admin-user.sh` is also a good option... As a kind of first checking whether the DB pod is up.  (Note that I had to add the Running pod test earlier there.)